cloud/scope/common.go

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
if this is used only for the tests, can be renamed to `common_test.go` or something else to restrict these functions only to the tests

Reply:
Renamed.